packages/pluggableWidgets/slider-web/src/ui/Slider.scss

```diff
@@ -6,9 +6,3 @@
         height: 100%;
     }
 }
-
-.widget-slider-no-value {
```

Comment:
Im afraid removing this class means that if users wants to use previous slider with new Atlas this class will be missing..